code/API_definitions/qos-profiles.yaml

```diff
@@ -86,6 +76,34 @@ paths:
                 type: array
                 items:
                   $ref: "#/components/schemas/QosProfile"
+        "400":
```

Comment:
Please copy the missing examples from quality-on-demand.yaml in #326

Reply:
@hdamker I've changed this to just use the Generic404 schema.  That will pick up the additional descriptions after both PRs are merged.